pkg/util/voljournal.go

```diff
@@ -323,8 +386,15 @@ Return values:
 	- string: Contains the UUID that was reserved for the passed in reqName
 	- error: non-nil in case of any errors
 */
-func (cj *CSIJournal) ReserveName(ctx context.Context, monitors string, cr *Credentials, pool, reqName, parentName, encryptionKmsConfig string) (string, error) {
-	var snapSource bool
+func (cj *CSIJournal) ReserveName(ctx context.Context, monitors string, cr *Credentials,
```

Comment:
in one way, the `pool` parameter now has more seggregation like `imagePool`, `journalPool`, `csiJounalPool` , imo, its better to document these concepts in source code comments for further use.

Reply:
Agree, will deal with it when amending the patch for further changes.

Reply:
Addressed in the latest update.